tests/apps/tests.py

```diff
@@ -275,14 +276,14 @@ class LazyA(models.Model):
 
         # Test models appearing twice, and models appearing consecutively
         model_keys = [('apps', model_name) for model_name in ['lazya', 'lazyb', 'lazyb', 'lazyc', 'lazya']]
-        apps.lazy_model_operation(test_func, *model_keys)
+        LazyA._meta.apps.lazy_model_operation(test_func, *model_keys)
```

Comment:
This is an example of a test that would benefit from being able to access `apps` as a passed argument.